pkg/rekor/identity.go

```diff
@@ -158,6 +167,21 @@ func MatchedIndices(logEntries []models.LogEntry, mvs MonitoredValues) ([]Identi
 					}
 				}
 			}
+
+			for _, monitoredOID := range mvs.OIDMatchers {
+				for _, cert := range certs {
+					match, err := oidMatchesPolicy(cert, monitoredOID.ObjectIdentifier, monitoredOID.ExtensionValues)
+					if err != nil {
+						return nil, fmt.Errorf("error with policy matching for UUID %s at index %d: %w", uuid, *entry.LogIndex, err)
+					}
+					if match {
+						matchedEntries = append(matchedEntries, IdentityEntry{
+							Index: *entry.LogIndex,
```

Comment:
Can we also update `IdentityEntry` to add fields to specify what OID it was matched on and what that OID's value was? This will be needed to use `IdentityEntry` to know why there was a match.